src/meta/src/stream/stream_graph/fragment.rs

```diff
@@ -128,6 +128,10 @@ impl BuildingFragment {
                 table.database_id = job.database_id();
                 table.schema_id = job.schema_id();
                 table.fragment_id = fragment_id;
+                #[cfg(not(debug_assertions))]
```

Comment:
This optimization will not take effect for MVs that have already been created in early version when upgrading.